xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-default/src/main/resources/ApplicationResources.properties

```diff
@@ -0,0 +1,82 @@
+# License along with this software; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+# 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+# ---------------------------------------------------------------------------
+
+###############################################################################
+# This contains the translations of the module in the default language
+# (generally English).
+#
+# Translation key syntax:
+#   <short top level project name>.<short module name>.<propertyName>
+#   where:
+#   * <short top level project name> = top level project name without the "xwiki-" prefix,
+#                                     for example: commons, rendering, platform, enterprise, manager, etc
+#   * <short module name> = the name of the Maven module without the <short top level project name> prefix,
+#                           for example: oldcore, scheduler, activitystream, etc
+#   * <propertyName> = the name of the property using camel case,
+#                      for example updateJobClassCommitComment
+#
+# Comments: it's possible to add some detail about a key to make easier to
+#   translate it by adding a comment before it. To make sure a comment is not
+#   assigned to the following key use at least three sharps (###) for the comment
+#   or after it.
+#
+# Deprecated keys:
+#   * when deleting a key it should be moved to deprecated section at the end
+#     of the file (between #@deprecatedstart and #@deprecatedend) and associated to the
+#     first version in which it started to be deprecated
+#   * when renaming a key, it should be moved to the same deprecated section
+#     and a comment should be added with the following syntax:
+#     #@deprecated new.key.name
+#     old.key.name=Some translation
+###############################################################################
+
+job.question.XClassBreakingQuestion.timeout=The action has been canceled because we have not received any answer after 5 minutes.
+job.question.XClassBreakingQuestion.confirm=Do you wish to continue?
+job.question.XClassBreakingQuestion.tree.selectAll=select all
+job.question.XClassBreakingQuestion.tree.selectNone=none
+job.question.XClassBreakingQuestion.tree.paginationNode={0} more....
+job.question.XClassBreakingQuestion.tree.freePages=Pages that do not contain any used XClass
+job.question.XClassBreakingQuestion.forbiddenConcernedPages=The following pages are incriminated in the forbidden action as they contain structured data types.
+
+job.question.XClassBreakingQuestion.refactoring/delete.title=You are about to delete pages that contain used XClass.
+job.question.XClassBreakingQuestion.refactoring/delete.explanation=If you delete these pages, some extensions might not work anymore.
+job.question.XClassBreakingQuestion.refactoring/delete.help=The recommended way of refactoring an XClass is to first delete or edit the objects that are using it.
```

Comment:
Sound weird written like this, it's not exactly a way. Something like "Its recommended to remove objects that are referring to an XClasses deleting it" ?